docs/user-guide/bestpractices.rst

```diff
@@ -24,6 +24,17 @@ Handling GPU Out-of-Memory (OOM) Errors
 NeMo Curator is designed to be scalable with large amounts of text data, but OOM errors occur when the available GPU memory is insufficient for a given task.
 To help avoid these issues and ensure efficient processing, here are some strategies for managing memory usage and mitigating OOM challenges.
 
+Controlling Partition Sizes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should consider using ``files_per_partition`` or ``blocksize`` when reading data. This can help reduce the memory load by processing large datasets in smaller chunks.
+
+#. ``blocksize`` argument is available for ``jsonl`` and ``parquet`` files. But for `parquet` files it's only available when ``add_filename=False``
```

Comment:
@praateekmahajan , Have we filed issues to `cuDF` to support these features long term. 

1. Supporting `add_file_name` to `cuDF` with multiple files (to decrease that perf delta)
2. Blocksize+`add_file_name` argument for `cuDF`.

Reply:
Discussed offline, i think there is confusion, we need to resurrect the issue https://github.com/dask/dask/issues/6575